python/py-tmuxp/Portfile

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=Portfile:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
we typically prefer Python ports from PyPI; in this case I guess you made the change because you want to install the documentation / man pages.

I personally am not sure if that's really necessary and it adds quite some dependencies, but I'll leave that to the maintainer.

Reply:
Dependencies are needed build-only. In most cases, the installation will be from binary archives. I added the examples directory yet. The documentation files may not be that important, but I use local documentation a lot.